README.md

```diff
@@ -140,12 +140,11 @@ Add the following step to your GitHub workflow (in example are used non-default
     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
   with:
     tag-name: "v0.1.0"
-    chapters: '[
-      {"title": "Breaking Changes 💥", "label": "breaking-change"},
-      {"title": "New Features 🎉", "label": "enhancement"},
-      {"title": "New Features 🎉", "label": "feature"},
-      {"title": "Bugfixes 🛠", "label": "bug"}
-    ]'
+    chapters:
+      - {"title": "Breaking Changes 💥", "label": "breaking-change"},
+      - {"title": "New Features 🎉", "label": "enhancement"},
+      - {"title": "New Features 🎉", "label": "feature"},
+      - {"title": "Bugfixes 🛠", "label": "bug"}
```

Comment:
See the difference between examples and choose your pattern you want to use. Your example is wrong.

```
chapters:
  - {"title": "Breaking Changes 💥", "label": "breaking-change"}
  - {"title": "New Features 🎉", "label": "enhancement"}
  - {"title": "New Features 🎉", "label": "feature"}
  - {"title": "Bugfixes 🛠", "label": "bug"}
```
This uses a YAML list (-) but also embeds JSON-like structures. However, the trailing commas in each line make it invalid YAML, as YAML does not permit trailing commas within lists or mappings.

```
chapters: |
  - {"title": "Breaking Changes 💥", "label": "breaking-change"},
  - {"title": "New Features 🎉", "label": "enhancement"},
  - {"title": "New Features 🎉", "label": "feature"},
  - {"title": "Bugfixes 🛠", "label": "bug"}
```
Interpretation: The | indicates that the value of chapters is a literal block scalar, which YAML treats as a multi-line string. However, this approach turns the content into a single string instead of a list, and it will not be parsed as a YAML list.
Result: The entire block would be treated as plain text, not a structured YAML list.

Reply:
There is no way how to provide array - the only supported types are string, number and boolean.